packages/table/test/tableTests.tsx

```diff
@@ -329,55 +329,81 @@ describe("<Table>", () => {
-                assertScrollPositionEquals(topLeftScrollContainer, 0, 0);
+                delayToNextAnimationFrame(() => {
+                    assertScrollPositionEquals(mainScrollContainer, SCROLL_OFFSET_X, SCROLL_OFFSET_Y);
+                    assertScrollPositionEquals(leftScrollContainer, 0, SCROLL_OFFSET_Y);
+                    assertScrollPositionEquals(topLeftScrollContainer, 0, 0);
+                    done();
+                });
             });
 
-            it("syncs quadrant scroll offsets when mouse-wheeling in the left quadrant", () => {
+            it("syncs quadrant scroll offsets when mouse-wheeling in the left quadrant", (done) => {
                 leftScrollContainer.scrollTop = SCROLL_OFFSET_Y;
                 TestUtils.Simulate.wheel(leftScrollContainer, {
                     deltaX: SCROLL_OFFSET_X,
                     deltaY: SCROLL_OFFSET_Y,
                 });
 
-                assertScrollPositionEquals(mainScrollContainer, SCROLL_OFFSET_X, SCROLL_OFFSET_Y);
-                assertScrollPositionEquals(topScrollContainer, SCROLL_OFFSET_X, 0);
-                assertScrollPositionEquals(topLeftScrollContainer, 0, 0);
+                delayToNextAnimationFrame(() => {
+                    assertScrollPositionEquals(mainScrollContainer, SCROLL_OFFSET_X, SCROLL_OFFSET_Y);
+                    assertScrollPositionEquals(topScrollContainer, SCROLL_OFFSET_X, 0);
+                    assertScrollPositionEquals(topLeftScrollContainer, 0, 0);
+                    done();
+                });
             });
 
-            it("syncs quadrant scroll offsets when mouse-wheeling in the top-left quadrant", () => {
+            it("syncs quadrant scroll offsets when mouse-wheeling in the top-left quadrant", (done) => {
                 TestUtils.Simulate.wheel(topLeftScrollContainer, {
                     deltaX: SCROLL_OFFSET_X,
                     deltaY: SCROLL_OFFSET_Y,
                 });
 
-                assertScrollPositionEquals(mainScrollContainer, SCROLL_OFFSET_X, SCROLL_OFFSET_Y);
-                assertScrollPositionEquals(topScrollContainer, SCROLL_OFFSET_X, 0);
-                assertScrollPositionEquals(leftScrollContainer, 0, SCROLL_OFFSET_Y);
+                delayToNextAnimationFrame(() => {
+                    assertScrollPositionEquals(mainScrollContainer, SCROLL_OFFSET_X, SCROLL_OFFSET_Y);
+                    assertScrollPositionEquals(topScrollContainer, SCROLL_OFFSET_X, 0);
+                    assertScrollPositionEquals(leftScrollContainer, 0, SCROLL_OFFSET_Y);
+                    done();
+                });
             });
         });
 
+        // Clock management
+        // ================
+
+        function delayToNextAnimationFrame(callback: () => void) {
+            // two rAFs are required to force the callback to execute on the *next* frame, allowing
+            // any initial rAF'd code to execute
```

Comment:
or one `setTimeout`.... why not use the built-in method that guarantees next-frame-execution?